specs/SelectSingle.spec.ts

```diff
@@ -52,4 +52,68 @@ describe('SelectSingle', () => {
 			})
 		})
 	})
+
+	describe('options', () => {
+		describe('prependLabel', () => {
+			const options = [
+				{ label: 'Item One', value: 'one' },
+				{ label: 'Item Two', value: 'two' },
+				{ label: 'Item Three', value: 'three' }
+			]
+			it('prepends the field label to the displayed option when attribute is set to true', async () => {
+				const myWrapper = mount({
+					data() { 
+						return { selectedOption: {}, options }},
+					template: `<div>
+									<SelectSingle :options="options" v-model="selectedOption" label="Sample SelectSingle" prependLabel="true"/>
```

Comment:
```suggestion
									<SelectSingle :options="options" v-model="selectedOption" label="Sample SelectSingle" :prependLabel="true"/>
```

Also fyi I see this warning when I run `yarn test`

`[Vue warn]: Invalid prop: type check failed for prop "prependLabel". Expected Boolean, got String with value "true".`